src/common/EIP712/TypedRequestData.ts

```diff
@@ -43,10 +46,24 @@ interface Types extends EIP712Types {
   RelayData: EIP712TypeProperty[]
 }
 
+// use these values in registerDomainSeparator
+export const GsnDomainSeparatorType = {
+  prefix: 'string name,string version',
+  name: 'GSN Relayed Transaction',
+  version: '2'
+}
+
+export function getRegisterDomainSeparatorData (): {prefix: string, dataPrefix: string } {
+  return {
+    prefix: GsnDomainSeparatorType.prefix,
+    dataPrefix: keccak256(GsnDomainSeparatorType.name) + keccak256(GsnDomainSeparatorType.version).slice(2)
```

Comment:
1) keccak returns a string with 0x, so here will be 0xaaa0xbbbb - how does it even work? decoeded as pure string?
2) why make version, name pre-hashed ?